src/main/java/spoon/pattern/internal/node/ForEachNode.java

```diff
@@ -54,7 +54,6 @@ public ForEachNode() {
 	@Override
 	public boolean replaceNode(RootNode oldNode, RootNode newNode) {
 		if (iterableParameter == oldNode) {
-			oldNode = newNode;
```

Comment:
I have to fix this one in another PR. The correct code is something like `itetrableParameter = newNode` ... but it needs a test and some type cast ...